src/components/NavBar.tsx

```diff
@@ -0,0 +1,24 @@
+import { NavLink } from "react-router-dom"
+import { getNavLinkClass } from "../types/utils/getNavLinkClass"
+
+export const NavBar = () => {
+  return (
+    <nav
+      data-cy="nav"
+      className="navbar is-fixed-top has-shadow"
+      role="navigation"
+      aria-label="main navigation"
+    >
+      <div className="container">
+        <div className="navbar-brand">
+          <NavLink className={getNavLinkClass} to="/">
```

Comment:
It seems like you're using the `getNavLinkClass` function directly as a string, which may not be what you intended. This function is supposed to be called with some parameters to return the correct class name. You might want to call this function and pass necessary parameters to it.